File: samples/client/petstore/flash/swagger/client/models/tag.py
package swagger.client/models {


    [XmlRootNode(name="Tag")]
    public class Tag {
    

    

        
        [XmlElement(name="id")]
        
        public var id: Number = None;

    

    

        
        [XmlElement(name="name")]
        
        public var name: String = None;

    

        public function toString(): String {
            var str: String = "Tag: ";
            
            str += " (id: " + id + ")";
            
            str += " (name: " + name + ")";
            
            return str;
        }


}
        

}

File: samples/client/petstore/dart/swagger/lib/model/tag.dart
part of swagger.api;

@Entity()
class Tag {
  
  @Property(name: 'id')
  int id = null;
  

  @Property(name: 'name')
  String name = null;
  
  Tag();

  @override
  String toString()  {
    return 'Tag[id=$id, name=$name, ]';
  }
}

